misc/python/materialize/cli/ci_annotate_errors.py

```diff
@@ -233,44 +266,76 @@ def annotate_logged_errors(log_files: list[str]) -> int:
 
                     if issue.info["number"] not in already_reported_issue_numbers:
                         unknown_errors.append(
-                            f"Potential regression [{issue.info['title']} (#{issue.info['number']}, closed)]({issue.info['html_url']}) in {linked_file}:  \n{formatted_error_message}"
+                            f"Potential regression [{issue.info['title']} (#{issue.info['number']}, closed)]({issue.info['html_url']}) in {location}:\n{formatted_error_message}"
                         )
                         already_reported_issue_numbers.add(issue.info["number"])
                     break
             else:
                 unknown_errors.append(
-                    f"Unknown error in {linked_file}:  \n{formatted_error_message}"
+                    f"Unknown error in {location}:\n{formatted_error_message}"
                 )
 
-    annotate_errors(
-        unknown_errors,
-        "Unknown errors and regressions in logs (see [ci-regexp](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/ci-regexp.md))",
-        "error",
-    )
-    annotate_errors(known_errors, "Known errors in logs, ignoring", "info")
+    for error in errors:
+        if isinstance(error, ErrorLog):
+            for artifact in artifacts:
+                if artifact["job_id"] == job and artifact["path"] == error.file:
+                    org = os.environ["BUILDKITE_ORGANIZATION_SLUG"]
+                    pipeline = os.environ["BUILDKITE_PIPELINE_SLUG"]
+                    build = os.environ["BUILDKITE_BUILD_NUMBER"]
+                    linked_file: str = f'[{error.file}](https://buildkite.com/organizations/{org}/pipelines/{pipeline}/builds/{build}/jobs/{artifact["job_id"]}/artifacts/{artifact["id"]})'
+                    break
+            else:
+                linked_file: str = error.file
+
+            handle_error(error.match, linked_file)
+
+        else:
+            if "in Code Coverage" in error.text:
```

Comment:
Are builds with code coverage excluded completely? I think that might not be a good idea.

Reply:
No, this just disabled checking for known issues in github, which makes no sense for the junit.xml from code coverage.

Reply:
Fixed my comment, I think we could completely exclude builds with coverage. Though, no strong opinion on it.